client/groups/filesystem/v1alpha1/client_generated.go

```diff
@@ -0,0 +1,57 @@
+// Code generated by csi-proxy-gen. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	"context"
+	"net"
+
+	"github.com/Microsoft/go-winio"
+	"google.golang.org/grpc"
+
+	"github.com/kubernetes-csi/csi-proxy/client"
+	pb "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1alpha1"
+	"github.com/kubernetes-csi/csi-proxy/client/apiversion"
+)
+
+const groupName = "filesystem"
+
+var version = apiversion.NewVersionOrPanic("v1alpha1")
+
+type wrapper struct {
+	client     pb.FileSystemClient
+	connection *grpc.ClientConn
+}
+
+// NewClient returns a client to make calls to the dummy API group version v1.
```

Comment:
comment nits: 
"dummy API group" => "Filesystem API group"
"version v1" => "version v1alpha1"

Reply:
Manual auto generation is hard ;)